lib/events.js

```diff
@@ -185,79 +180,65 @@ api.aggregateHistory = callbackify(async ({
     {$unwind: '$_parents'},
     {$replaceRoot: {newRoot: '$_parents'}},
   ];
-  const startTime = Date.now();
+  const timer = new _cache.Timer();
```

Comment:
Is there a reason why this is not using a constructor arrangement that passes `name` ledgerNodeId`, instead of passing those values into the `start` API?

Reply:
To allow reuse of the timer. It could have gone either way.